### PR TITLE
Add default workflow permissions for enterprise/org/repo

### PR DESCRIPTION
**Pull request by ErikElkins**

### CODE REVIEW

File: github/actions_permissions_enterprise.go [hunk 1]
```diff
@@ -29,6 +29,14 @@ func (a ActionsPermissionsEnterprise) String() string {
 	return Stringify(a)
 }
 
+// DefaultWorkflowPermissionEnterprise represents the default permissions for GitHub Actions workflows
```
Comment (gmlewis) [inline github/actions_permissions_enterprise.go:32]:
```suggestion
// DefaultWorkflowPermissionEnterprise represents the default permissions for GitHub Actions workflows for an enterprise.
```

File: github/actions_permissions_enterprise.go [hunk 2]
```diff
@@ -205,3 +213,46 @@ func (s *ActionsService) EditActionsAllowedInEnterprise(ctx context.Context, ent
...
+
+	return permissions, resp, nil
+}
+
+// EditDefaultWorkflowPermissionsInEnterprise sets the GitHub Actions default workflow permissions for an enterprise.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/actions/permissions#set-default-workflow-permissions-for-an-enterprise
+//
+//meta:operation PUT /enterprises/{enterprise}/actions/permissions/workflow
+func (s *ActionsService) EditDefaultWorkflowPermissionsInEnterprise(ctx context.Context, enterprise string, defaultWorkflowPermissionEnterprise DefaultWorkflowPermissionEnterprise) (*DefaultWorkflowPermissionEnterprise, *Response, error) {
```
Comment (gmlewis) [inline github/actions_permissions_enterprise.go:244]:
```suggestion
func (s *ActionsService) EditDefaultWorkflowPermissionsInEnterprise(ctx context.Context, enterprise string, permissions DefaultWorkflowPermissionEnterprise) (*DefaultWorkflowPermissionEnterprise, *Response, error) {
```

File: github/actions_permissions_orgs.go [hunk 3]
```diff
@@ -42,6 +42,14 @@ func (a ActionsAllowed) String() string {
 	return Stringify(a)
 }
 
+// DefaultWorkflowPermissionOrganization represents the default permissions for GitHub Actions workflows
```
Comment (gmlewis) [inline github/actions_permissions_orgs.go:45]:
```suggestion
// DefaultWorkflowPermissionOrganization represents the default permissions for GitHub Actions workflows for an organization.
```

File: github/repos_actions_permissions.go [hunk 4]
```diff
@@ -23,13 +23,22 @@ func (a ActionsPermissionsRepository) String() string {
 	return Stringify(a)
 }
 
+// DefaultWorkflowPermissionRepository represents the default permissions for GitHub Actions workflows
```
Comment (gmlewis) [inline github/repos_actions_permissions.go:26]:
```suggestion
// DefaultWorkflowPermissionRepository represents the default permissions for GitHub Actions workflows for a repository.
```

File: github/actions_permissions_orgs.go [hunk 5]
```diff
@@ -218,3 +226,46 @@ func (s *ActionsService) EditActionsAllowed(ctx context.Context, org string, act
...
+
+	return permissions, resp, nil
+}
+
+// EditDefaultWorkflowPermissionsInOrganization sets the GitHub Actions default workflow permissions for an organization.
+//
+// GitHub API docs: https://docs.github.com/rest/actions/permissions#set-default-workflow-permissions-for-an-organization
+//
+//meta:operation PUT /orgs/{org}/actions/permissions/workflow
+func (s *ActionsService) EditDefaultWorkflowPermissionsInOrganization(ctx context.Context, org string, defaultWorkflowPermissionOrganization DefaultWorkflowPermissionOrganization) (*DefaultWorkflowPermissionOrganization, *Response, error) {
```
Comment (gmlewis) [inline github/actions_permissions_orgs.go:257]:
```suggestion
func (s *ActionsService) EditDefaultWorkflowPermissionsInOrganization(ctx context.Context, org string, permissions DefaultWorkflowPermissionOrganization) (*DefaultWorkflowPermissionOrganization, *Response, error) {
```

File: github/repos_actions_permissions.go [hunk 6]
```diff
@@ -64,3 +73,46 @@ func (s *RepositoriesService) EditActionsPermissions(ctx context.Context, owner,
...
+
+	return permissions, resp, nil
+}
+
+// EditDefaultWorkflowPermissions sets the GitHub Actions default workflow permissions in a repository.
+//
+// GitHub API docs: https://docs.github.com/rest/actions/permissions#set-default-workflow-permissions-for-a-repository
+//
+//meta:operation PUT /repos/{owner}/{repo}/actions/permissions/workflow
+func (s *RepositoriesService) EditDefaultWorkflowPermissions(ctx context.Context, owner, repo string, defaultWorkflowPermissionRepository DefaultWorkflowPermissionRepository) (*DefaultWorkflowPermissionRepository, *Response, error) {
```
Comment (gmlewis) [inline github/repos_actions_permissions.go:104]:
```suggestion
func (s *RepositoriesService) EditDefaultWorkflowPermissions(ctx context.Context, owner, repo string, permissions DefaultWorkflowPermissionRepository) (*DefaultWorkflowPermissionRepository, *Response, error) {
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @ErikElkins!
Just a few minor tweaks, please, and then we will be ready for a second LGTM+Approval from any other contributor to this repo before merging.

File: github/repos_actions_permissions.go (hunk 6 above)
Comment (ErikElkins) [reply github/repos_actions_permissions.go]: Updated the variable name!

**Review by ecrupper — APPROVED** [review]

File: github/actions_permissions_orgs.go (hunk 5 above)
Comment (ErikElkins) [reply github/actions_permissions_orgs.go]: I've updated the variable name!

**Review by gmlewis — APPROVED** [review]
Thank you, @ErikElkins and @ecrupper !
LGTM.
Merging.

**Pull request merged**